mrpt_localization/include/mrpt_localization_node.h

```diff
@@ -47,8 +47,7 @@
 #include <mrpt/version.h>
 #if MRPT_VERSION>=0x130
```

Comment:
Since the `master` branch in these ROS modules is now aimed at mrpt 2.*, there is no need to keep these `#if MRPT_VERSION>=xxx` pragmas. All of them can be removed.